aten/src/ATen/native/cuda/MemoryAccess.cuh

```diff
@@ -191,6 +224,38 @@ struct vectorized {
       to_[index] = v;
     }
   }
+
+  template <typename scalar_t>
+    __device__ inline void store_result(int i, scalar_t f_return, scalar_t *results) {
+      results[i] = f_return;
+    }
+};
+
+template <typename data_t, typename inp_calc_t, typename out_calc_t, int num_outputs>
+struct multi_outputs_unroll : unroll<data_t, inp_calc_t, out_calc_t, num_outputs> {
+
+  __device__ multi_outputs_unroll(data_t data, int remaining, inp_calc_t ic, out_calc_t oc):
+    unroll<data_t, inp_calc_t, out_calc_t, num_outputs>({data, remaining, ic, oc}) {}
+
+  template <typename return_t>
+  __device__ inline void store(return_t *from, int idx) {
```

Comment:
what is `store` and what is `store_result`?

Reply:
`store` does store the results from `* from` to `data`. `store_result` stores the return value from `func_t f` to `*results` which is `return_t results[thread_work_size]`.

Reply:
In short, `store`'s functionality is not changed at all in this PR.